Sources/MongoSwift/BSON/BSONDecoder.swift

```diff
@@ -164,6 +164,20 @@ public class BSONDecoder {
                                   debugDescription: "Unable to parse JSON string \(json)"))
     }
 
+    /// Internal function used to  convert encountered `DecodingError`s to `.internalErrors`. Use this when using the
```

Comment:
the first sentence reads like this is something that takes in a `DecodingError` and returns an `internalError`. can we swap the sentences or rephrase so it's clearer that it is basically just `decode` with a different error type?

Reply:
Fixed. Now reads:
>Internal version of `decode` that throws `.internalErrors` instead of `DecodingErrors`. Use this when using the decoder internally on non-user-modified types.